src/components/WizardFooter/WizardFooter.stories.mdx

```diff
@@ -0,0 +1,150 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs'
```

Comment:
Can I ask you to convert this to .tsx instead of .mdx? The reason we prefer .tsx is so that we get type safety in storybook code as well.

Reply:
Done!